### PR TITLE
format json output

### PR DESCRIPTION
**Pull request by like-inspur**
use jq to format json output

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -430,7 +430,7 @@ The query API allows accessing pushed metrics and build and runtime information.
 
 * For example :
 
-        curl -X GET http://pushgateway.example.org:9091/api/v1/status
+        curl -X GET http://pushgateway.example.org:9091/api/v1/status|jq
```
Comment (beorn7) [inline README.md:433]: Good idea, but could you separate the pipe symbol with spaces around it to make this more readable? I.e. `curl -X GET http://pushgateway.example.org:9091/api/v1/status | jq`

**Comment by roidelapluie** [comment]
In general I recommend `|python -m json.tool` instead of jq as it is more widespread.

**Comment by beorn7** [comment]
I'd still go with `jq` here as it keeps the example short. It's just an example after all.

I'd rather not start a discussion what is really more widespread and if you should better say `python3` and what not.

File: README.md (hunk 1 above)
Comment (like-inspur) [reply README.md]: I have modify and commit again

**Comment by like-inspur** [comment]
> In general I recommend `|python -m json.tool` instead of jq as it is more widespread.
From current output, use jq match current output like below and jq output has color
```
root@mgt01:~# curl -X GET http://10.110.19.60:9091/api/v1/status | python -m json.tool
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   599  100   599    0     0   194k      0 --:--:-- --:--:-- --:--:--  194k
{
    "data": {
        "build_information": {
            "branch": "HEAD",
            "buildDate": "20200311-18:51:01",
            "buildUser": "root@0e823ccfff84",
            "goVersion": "go1.13.8",
            "revision": "b7e0167e9574f4f88404dde9653ee1d3c940f2eb",
            "version": "1.2.0"
        },
        "flags": {
            "log.format": "logfmt",
            "log.level": "debug",
            "persistence.file": "/pushgateway/data",
            "persistence.interval": "5m0s",
            "push.disable-consistency-check": "false",
            "web.enable-admin-api": "true",
            "web.enable-lifecycle": "true",
            "web.external-url": "",
            "web.listen-address": ":9091",
            "web.route-prefix": "",
            "web.telemetry-path": "/metrics"
        },
        "start_time": "2020-05-08T08:38:00.624651246Z"
    },
    "status": "success"
}
```
```
root@mgt01:~# curl -X GET http://10.110.19.60:9091/api/v1/status | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   599  100   599    0     0   292k      0 --:--:-- --:--:-- --:--:--  292k
{
  "status": "success",
  "data": {
    "build_information": {
      "branch": "HEAD",
      "buildDate": "20200311-18:51:01",
      "buildUser": "root@0e823ccfff84",
      "goVersion": "go1.13.8",
      "revision": "b7e0167e9574f4f88404dde9653ee1d3c940f2eb",
      "version": "1.2.0"
    },
    "flags": {
      "log.format": "logfmt",
      "log.level": "debug",
      "persistence.file": "/pushgateway/data",
      "persistence.interval": "5m0s",
      "push.disable-consistency-check": "false",
      "web.enable-admin-api": "true",
      "web.enable-lifecycle": "true",
      "web.external-url": "",
      "web.listen-address": ":9091",
      "web.route-prefix": "",
      "web.telemetry-path": "/metrics"
    },
    "start_time": "2020-05-08T08:38:00.624651246Z"
  }
}
```

**Comment by beorn7** [comment]
I guess @drumilpatel2000 used `jq`, too, to create the example output.

Whatever we use here, it's really not a big deal. `jq` is quite well known, and `python` is almost everywhere, of course. It's really not worth a huge discussion.

I'll merge this as is. Thank you very much, everyone, for your input.

**Pull request merged**